uoft-tapp/tapp
# [Backend] Seed data

Please see issue #166 for more detail.

**Please merge this PR after PR #182**

To test this PR:
1) `docker-compose up --build`
2) `docker-compose exec api rake db:drop db:create db:migrate`
3) `docker-compose exec api rake db:seed`: There should be a log that tells you if the seeding succeeded or not
4) use browser to check db state via GET routes
6) change the content in `api/db/seed/mock_data.json`
5) `docker-compose exec api rake db:seed`: The db should update to match the changed `mock_data`
7) use browser to check db state via GET routes

**Note:** The last line in `seed.rb` generates fake data. You can comment it out if you only want to deal with the data from the `mock_data.json`.
